vgen/src/main/resources/io/gapi/vgen/py/discovery_fragment.snip

```diff
@@ -0,0 +1,187 @@
+            request.execute()
+        @else
+            response = request.execute()
+            
+            @# {@TODO()} Change code below to process the 'response' dict:
+            @let json = importHandler.addImportStandard("json"), \
+                 _ = importHandler.addImportStandard("__future__", "print_function")
+                print({@json}.dumps(response))
+            @end
+        @end
+    @end
+@end
+
+# TODO(tcoffee): configure snippet grouping to handle long argument lists
+@private arguments(method)
+    @if context.hasRequestField(method)
+        @let params = context.getFlatMethodParams(method)
+            @if params
+                {@argumentList(params)}, {@requestName(method)}
+            @else
+                {@requestName(method)}
+            @end
+        @end
+    @else
+        {@argumentList(context.getMethodParams(method))}
+    @end
+@end
+
+# TODO(tcoffee): generate appropriately typed default values for params
+@private initialize(method, params)
+    @let methodName = context.getMethodName(method), \
+         apiaryConfig = context.getApiaryConfig, \
+         signatureType = apiaryConfig.getType(method.getRequestTypeUrl), \
+         params = context.getFlatMethodParams(method)
+        @if params
+      
+            @# {@TODO()} Change placeholders below to values for parameters to the '{@methodName}' method:
+            
+        @end
+        @join param : params
+            @let paramField = context.getField(signatureType, param), \
+                 paramValue = context.typeDefaultValue(signatureType, paramField), \
+                 paramDescription = apiaryConfig.getDescription(signatureType.getName, param)
+                {@item(paramDescription)}
+                {@param} = {@paramValue}
+            @end
+        @end
+    @end
+@end
+
+# TODO(tcoffee): add additional commented code for individual fields of request objects?
+@private initializeRequest(method)
+    {@requestName(method)} = {
+    @if context.isPatch(method)
+        @# {@TODO()} Add desired entries of the '{@requestName(method)}' dict to be changed
+    @else
+        @# {@TODO()} Add desired entries of the '{@requestName(method)}' dict
+    @end
+    }
+    
+@end
+
+@private serviceName()
+    service
+@end
+
+@private requestName(method)
+    {@context.getSampleVarName(context.getRequestField(method).getTypeUrl)}
+@end
+
+@private item(description)
+    @if description
+        @# * {@description}
+    @end
+@end
+
+@private elementPath(elements)
+    @join element : elements on "."
+        {@element}()
+    @end
+@end
+
+@private argumentList(arguments)
+    @join argument : arguments on ", "
+        {@argument}
+    @end
+@end
+
+@private TODO()
+    {@context.getTODO()}
+@end
+
+@private alwaysImport() fill
+  {@context.silent(importHandler.addImportLocal("apiclient", "discovery"))}
+  {@context.silent(importHandler.addImportLocal("oauth2client.client", "GoogleCredentials"))}
```

Comment:
No, the naming is different in Python (from Java) to be consistent with what was done for Gapic. In Java, typeName returns names for referencing imports, and addImport silently wraps typeName. In Python, the "addImport" methods are analogous to typeName (to follow what was done before), and since there are multiple such methods, it seemed more convenient to have an independent silencing wrapper.